tests/cloudpickle_test.py

```diff
@@ -614,13 +614,33 @@ def test_NotImplementedType(self):
         res = pickle_depickle(type(NotImplemented), protocol=self.protocol)
         self.assertEqual(type(NotImplemented), res)
 
-    def test_builtin_function_without_module(self):
-        on = object.__new__
-        on_depickled = pickle_depickle(on, protocol=self.protocol)
-        self.assertEqual(type(on_depickled(object)), type(object()))
+    @pytest.mark.skipif(
+        sys.version_info[0] >= 3,
+        reason="builtin_function_or_method are special-cased only in python2")
```

Comment:
Why skip this test under Python 3? This test should should also pass under Python 3, no? If so just add a comment that this test is only necessary to test the support of pickling of builtin functions in Python 2.